### ISSUE TITLE
Unable to name attribute 'type' in model.

### ISSUE THREAD

**Issue by mbret** [balderdashy/sails#2322]
Hi, it seems that using 'type' as name of a model attribute is not working.
At least with memory / disk / mysql adapters

Ex:
module.exports ={
  attributes: {
      type: {}
  }
}

I guess it's because 'type' keyword is used by waterline but in my project case I would have been able to use type as a field name. Does it's expected behaviour ?

Thanks

**Comment by CWyrtzen** [comment]
@mbret Is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @mbret.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
